group_vars/location_gubener/snmp.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+snmp_devices:
+  - hostname: gubener-sama-5ghz
+    address: 10.31.77.160
+    snmp_profile: airos_8
```

Comment:
Furthermore you'd need to make sure that SNMP with community `public` is enabled on the device. The next step would be to build an image for gubener-core and then flash it using `sysupgrade $IMAGE`

Best Simon

Reply:
I have put you an example picture into the comments how to enable it. :)